app/views/books/show.html.erb

```diff
@@ -0,0 +1,9 @@
+Book Title: <%= @book.title %>
+<% ratings = @book.ratings %>
+<% average_ratings = ratings.map(&:rating).sum/ratings.count %>
+<% highest_rating = ratings.max %>
+<% lowest_rating = ratings.min %>
+Average Rating: <%= average_ratings.round(4) %>
+Highest Rating: <%= highest_rating.rating %>
+                <%= highest_rating.body %>
+Lowest Rating:  <%= lowest_rating.rating %>
```

Comment:
All this logic should live on an instance of Book. Any time you feel the need to save a variable in your view, it should be a bright red flag.